scripts/update_e2e_coverage.py

```diff
@@ -98,12 +98,12 @@ def get_test_targets(test_suite_path):
   # Check if the suite exists (which may not be true for failing suites)
   target_dir = test_suite.split(':')[0]
   query = ['bazel', 'query', f'{target_dir}/...']
-  targets = subprocess.check_output(query, stderr=subprocess.DEVNULL)
+  targets = subprocess.check_output(query)
```

Comment:
The output of this is pretty annoying when things do work, but it's mostly run by an automated script anyway.
```
Starting local Bazel server and connecting to it...
Loading: 0 packages loaded
Loading: 2 packages loaded
    currently loading: integrations/tensorflow/e2e
Loading: 3 packages loaded
Loading: 3 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
```

Reply:
Yep. We don't care this much normally; but if something went wrong then the output can be quite useful.